packages/subgraph/subgraph.yaml

```diff
@@ -8,7 +8,7 @@ dataSources:
     source:
       address: "0xae6ADd894F8a1BcAC10b153dc59Cab1da9656836"
       abi: DataEdge
-      startBlock: 12052156
+      # startBlock: 1205215  # Hardhat starts at block zero.
```

Comment:
This will break production deployment.

I'm not sure how we should handle this change for deploying to a local testnet. Does Hardhat or Ganache accepts "starting" from a late block?